controllers/user.js

```diff
@@ -26,6 +26,15 @@ const updateUser = async (req, res) => {
   res.status(204).end()
 }
 
+const updateUserProfile = async (req, res) => {
+  const { id } = req.params
+  const updateData = req.body
+
+  await userService.updateUserProfile(id, updateData)
+
+  res.status(201).end()

```

Comment:
i thnk 201 status (created) is not suitable in this case, change it to 204 if you don't send back any data

Reply:
yeap, already changed